pgjdbc/src/main/java/org/postgresql/jdbc/PgArray.java

```diff
@@ -161,6 +162,15 @@ public Object getArrayImpl(long index, int count, Map<String, Class<?>> map) thr
       throw org.postgresql.Driver.notImplemented(this.getClass(), "getArrayImpl(long,int,Map)");
```

Comment:
should we support this too? you would need only to convert map -> json and then use your implementation.